lib/common/client.ts

```diff
@@ -1,5 +1,5 @@
 import { useState, useEffect } from "react";
-import type { TUseStoryblokState } from "../types";
+import { TUseStoryblokState } from "../types";
```

Comment:
Why is the `type` keyword removed?

Reply:
I think this is not needed, as this is already a type.